v4/config/source/nats/options.go

```diff
@@ -16,6 +16,16 @@ type (
 
 // WithUrl sets the nats url.
 func WithUrl(a string) source.Option {
```

Comment:
Would it be better to just do `WithUrl(a ...string)` which would allow setting multiple urls and maintain backwards compatibility?

Reply:
Yes, it will be better.